docs/layers.md

```diff
@@ -665,30 +665,57 @@ At zoom 12 and below, we consolidate some landuse kinds to reduce the amount of
 
 The current mappings are:
 
+* `airfield` -> `aerodrome`
 * `allotments` -> `urban_area`
+* `artwork` -> `urban_area`
+* `attraction` -> `urban_area`
 * `bare_rock` -> `desert`
-* `cemetery` -> `urban_area`
+* `cinema` -> `urban_area`
+* `college` -> `university`
 * `commercial` -> `urban_area`
+* `common` -> `grassland`
+* `container_terminal` -> `urban_area`
+* `cutline` -> `forest`
+* `dam` -> `land`
```

Comment:
To `barren`

Reply:
Ooops, I got confused by your comment:

> "land" but also updated min_zoom to 11+

And thought you wanted it remapped to "land".

Fixed in 9c9d174.